formulas/wick-base/README.md

```diff
@@ -492,6 +504,57 @@ Examples
 Returns nothing.
 
 
+`wickServiceAddDependency()`
+----------------------------
+
+Internal: Makes the service provided depend on another service which could change the order services are loaded in at boot time.
+
+* $1 - The name of the service you wish to add a dependency to.
+* $2 - The name of the service you wish to add as a dependency to the service.
+
+Returns: 1 if $1 is not provided and 2 if $2 is not provided.
+
+
+`wickServiceAddDependencyIfEnabled()`
+-------------------------------------
+
+Internal: Adds the dependency to the service if that dependency is enabled.
+
+* $1 - The service to add the dependency to
+* $2 - The service name you would like to add as a dependency to $1
+
+Returns nothing.
```

Comment:
Doesn't this return 1 or 2 if $1 or $2 are empty?  Or you could say it returns the result of `wickServiceAddDependency` if the service is enabled.